addons/l10n_ar/views/report_invoice.xml

```diff
@@ -239,6 +239,14 @@
 
             </div>
         </xpath>
+
+        <!-- RG 5003: Add legend for 'A' documents that have a Monotribuista receptor -->
+        <p name="comment" position="after">
+            <p t-if="o.partner_id.l10n_ar_afip_responsibility_type_id.code in ['6', '13'] and o.l10n_latam_document_type_id.l10n_ar_letter == 'A'" >
+                El crédito fiscal discriminado en el presente comprobante, sólo podrá ser computado a efectos del Régimen de Sostenimiento e Inclusión Fiscal para Pequeños Contribuyentes de la Ley Nº 27.618
+            </p>
+        </p>
```

Comment:
Keep it in English and update the `pot`

Reply:
Hi @william-andre
Thank you for the review.

> Seems good. Would be great if I could have a functional validation, I have no idea where/how to find the docs with the information you provided. (JCO is not available this week)

Indeed, @vbe-odoo gives us a functional review +1 on the Latam instance. Valentino, please could you give us functional +1 in this PR on Github just to have the record that has been functionally validated? We know Jos is not here this week that is the reason why we are asking you. We have a deadline for this RG so we need to merge it this week.

> Keep it in English and update the pot

Not sure what to do here. This paragraph only appears in local invoices and is an extract from an RG that as far as I know requires that exact text: that is the reason we leave it in Spanish directly and we do not add any English translation. What do you think about this @MarianaLuzTorres  @jjscarafia?